eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/src/cc/codechecker/plugin/config/CodeCheckerContext.java

```diff
@@ -159,7 +163,7 @@ public void setActiveEditorPart(IEditorPart partRef,boolean refresh) {
             for (IViewReference vp : page.getViewReferences()) {
                 if (vp.getId().equals(ReportListView.ID)) {
                     ReportListView rlv = (ReportListView) vp.getView(true);
-                    if (rlv.getViewerRefresh()) {
+                    if (rlv != null && rlv.getViewerRefresh()) {
```

Comment:
remove rlv!=null check if rlv really cannot be null. Throw and exception if it can be null